### ISSUE TITLE
type PDESystem has no field analytic

### ISSUE THREAD

**Issue by bgctw** [SciML/MethodOfLines.jl#348]
**Describe the bug 🐞**

When experimenting with [minimal required versions](https://github.com/cjdoris/julia-downgrade-compat-action),
 I encountered error during precompilation: "type PDESystem has no field analytic". 

**Expected behavior**

Precompilation should succeed.

**Additional context**

The `analytic` field was added to type `PDESystem` in package `ModelingToolkit` at version 6.48. 
Hence, the `Project.toml` in `MethodOfLines` should set its minimal version requirement to at least 6.48. Currently, its at "6".

**Comment by ChrisRackauckas** [comment]
Yes indeed, sorry about that. We're going around and adding downgrade CI to fix up lower bounds https://github.com/SciML/NonlinearSolve.jl/pull/338. We just haven't gotten that to this project yet.
